ArtOfIllusion/src/artofillusion/PreferencesWindow.java

```diff
@@ -168,10 +168,11 @@ void processEvent()
     });
     toolChoice.setSelectedIndex(prefs.getUseCompoundMeshTool() ? 1 : 0);
     localeChoice = new BComboBox();
+    Locale defaultLocale = Locale.getDefault();
     Locale languages[] = Translate.getAvailableLocales();
     for (int i = 0; i < languages.length; i++)
     {
-      localeChoice.add(languages[i].getDisplayName(prefs.getLocale()));
+      localeChoice.add(languages[i].getDisplayName(defaultLocale));
```

Comment:
Does this mean that no matter what language you select, the list of available languages will always be displayed in the one returned by `Locale.getDefault()`?

Reply:
Yes, it does. Not sure what I think about that. It seems orthagonal to the purpose of this PR, in any case.

Reply:
I really don't like this one. I'd prefer the locale options to be listed in the selected language. If we really wanted to get fancy, they would each be rendered in their own locale, but that would be a whole different feature discussion.

Reply:
>I really don't like this one. I'd prefer the locale options to be listed in the selected language.

So if Your cat accidentally switch locale to ex, Japan You prefer to see next time languages list in Japan locale and try to understand which hieroglyphs line is English rather than see it in host default locale?

Reply:
If your cat manages that, she needs her own user account... :)

Seriously, though, I'd _prefer_ that each language choice was rendered _in that language_, which is way out of scope for this PR. Your proposed change here is also out of scope for fixing the bug that you found, so does not belong in this PR.